src/test/test-quota-util.c

```diff
@@ -0,0 +1,71 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <sys/mman.h>
+#include <sys/quota.h>
+
+#include "blockdev-util.h"
+#include "device-util.h"
+#include "quota-util.h"
+#include "tests.h"
+
+#define NUMINTS  (1000)
+#define FILESIZE (NUMINTS * sizeof(int))
+
+int *file_nmapped_addr(const char *path);
+
+TEST(quotactl_devnum) {
+        _cleanup_free_ char *devnode = NULL;
+        int r;
+        int fd;
+        dev_t devnum;
+        int *m_addr;
+        const char *path = "/tmp/mmapped.bin";
```

Comment:
Please do not use fixed path.
Create a temporary file or directory, e.g. by mkdtemp_malloc().
See other test codes that uses the function.